addons/website_sale/controllers/main.py

```diff
@@ -872,6 +872,7 @@ def payment_get_status(self, sale_order_id, **post):
                 values.update({
                     'tx_ids': True,
                     'state': state,
+                    'order': order,
```

Comment:
@rmu-odoo 

I think, you need acquirer_id instead of order object. so just pass acquirer_id into template.